### PR TITLE
Add flag to specify the number of cpus

### PR DESCRIPTION
**Pull request by pvdrz**
Apparently you can't rename a branch from github's website without it closing all your PRs with that branch. So this is  just #2545

### CODE REVIEW

**Review by oli-obk — COMMENTED** [review]
Lgtm, but needs docs instead of FIXMEs

**Comment by RalfJung** [comment]
@bors r-
still needs documentation in the readme

File: README.md [hunk 1]
```diff
@@ -324,6 +324,9 @@ environment variable. We first document the most relevant and most commonly used
   ensure alignment.  (The standard library `align_to` method works fine in both modes; under
   symbolic alignment it only fills the middle slice when the allocation guarantees sufficient
   alignment.)
+* `-Zmiri-num-cpus` states the number of available CPUs to be reported by miri. By default, the
```
Comment (RalfJung) [inline README.md:327]: please keep the list alphabetically sorted

**Comment by RalfJung** [comment]
Thanks. :)
@bors r+
(bors is on strike unfortunately, so not sure when this will actually land)

**Comment by pvdrz** [comment]
yay @bors is back! :tada:

**Pull request merged**